module/zfs/dmu_send.c

```diff
@@ -69,6 +69,11 @@ int zfs_send_set_freerecords_bit = B_TRUE;
 static char *dmu_recv_tag = "dmu_recv_tag";
 const char *recv_clone_name = "%recv";
 
+/*
+ * Use this to override the recordsize calculation for fast zfs send estimates.
+ */
+uint64_t zfs_override_estimate_recordsize = 0;
```

Comment:
I noticed that the illumos port of this commit changed this tunable's default to 0 (from 8192, in the original commit delphix/delphix-os@d40bd0365a398c0875cda2851b7204e693d049f4 ), which means the behavior doesn't change unless you reach out and twiddle this. 

Would it make sense to default to this behavior being enabled, or is there a reason to disable it?